packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -310,6 +311,19 @@ class SnackBar extends StatefulWidget {
   /// circular corner radius of 4.0.
   final ShapeBorder? shape;
 
+  /// Defines how snackbar area will react to user hits (including margin)
```

Comment:
"Defines how the snack bar area, including margin, will behave during hit testing."

Reply:
Thanks for the corrections, my English isn't perfect for competent writing of documentation, I will correct it!